pkg/tests/BUILD

```diff
@@ -31,8 +31,8 @@ filegroup(
         ":test_tar_package_dir_file",
     ],
     visibility = [
-        "//tests:__subpackages__",
         "//legacy/tests:__subpackages__",
+        "//tests:__subpackages__",
```

Comment:
Did you do this with `buildifier`, or by hand?  Might be worth calling out `buildifier` in your commit message if done with `buildifier`.